.github/workflows/osrm-backend.yml

```diff
@@ -157,7 +157,7 @@ jobs:
             runs-on: ubuntu-20.04
             BUILD_TOOLS: ON
             BUILD_TYPE: Release
-            CLANG_VERSION: 11.0.0
+            CLANG_VERSION: 11
```

Comment:
Do we have any obligations to support certain old compilers btw?

Reply:
Ideally for release builds we want to use a compiler that supports the minimum requirements of the oldest Node platform in our matrix. This is to avoid any ABI/stdlib incompatibilities with the host environments (example: #4567)

This is the toolchain info for Node 12: https://github.com/nodejs/node/blob/v12.x/BUILDING.md